### PR TITLE
langsmith 0.0.39

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**Current package info**

| Channel | Downloads | Version | Platforms | License |
| --- | --- | --- | --- | --- |
| conda-forge | [Conda Downloads](https://anaconda.org/conda-forge/langsmith) | [Conda Version](https://anaconda.org/conda-forge/langsmith) | [Conda Platforms](https://anaconda.org/conda-forge/langsmith) | Conda License |

Changelog: https://github.com/langchain-ai/langsmith-sdk/releases
License: https://github.com/langchain-ai/langsmith-sdk/blob/v0.0.39/LICENSE
Requirements: https://github.com/langchain-ai/langsmith-sdk/blob/v0.0.39/python/pyproject.toml

Actions:
1. Clean up noarch python
2. Add `--no-deps --no-build-isolation` to script
3. Update in run: pydantic >=1.0.0,<3.0.0
4. Fix doc_url

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -46,7 +45,7 @@ about:
   license: MIT
   license_family: MIT
   license_file: LICENSE
-  doc_url: https://docs.smith.langchain.com/docs/
+  doc_url: https://docs.smith.langchain.com/
   dev_url: https://github.com/langchain-ai/langchainplus-sdk
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
  dev_url: https://github.com/langchain-ai/langsmith-sdk
```

**Review by cbouss — APPROVED** [review]
LGTM, just the dev_url to update

File: recipe/meta.yaml [hunk 2]
```diff
@@ -6,24 +6,23 @@ package:
...
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
   number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
     - langsmith = langsmith.cli.main:main
 
 requirements:
   host:
-    - python >=3.8,<4.0
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:21]: skip: True  # [py<38] ?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -6,24 +6,23 @@ package:
...
+    - python
     - poetry-core
     - pip
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:22]: wheel ?

File: recipe/meta.yaml (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:21]: No need. We build py>=38 by default. Only if there is a difference then we can skip

File: recipe/meta.yaml (hunk 3 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:22]: poetry-core has its own wheel builder

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**